app/services/true_layer/api_client.rb

```diff
@@ -21,8 +21,8 @@ def accounts
 
     def transactions(account_id, date_from, date_to)
       params = {
-        from: date_from.to_date.iso8601,
-        to: date_to.to_date.iso8601
+        from: date_from.to_datetime.iso8601,
+        to: date_to.to_datetime.iso8601
       }
```

Comment:
This will give a string like `2020-06-16T00:00:00+01:00`, whereas the TrueLayer documentation referenced in the ticket asks for `2018-02-18T00:00:00`.  I was actually wondering whether or not we needed to take timezones into account - but I guess the answer is no - TrueLayer always assumes London timezone with BST where applicable.

Reply:
I have changed the date procuded to `.to_datetime.utc` this results in `yyyy-mm-ddThh:mm:ssZ` without additional timezone data.
It doesn't specifically match the true layer documentation but they return values suffixed with `Z` so I have made the assumption that they will accept them on the inputs.
I have tested this against the TrueLayer API and it successfully returns transaction data.